azure/azure-how-to/instances/create-a-golden-image-pipeline.rst

```diff
@@ -0,0 +1,274 @@
+Create an Ubuntu golden image pipeline
+======================================
+
+This guide will provide instructions for using GitHub Actions to create a pipeline for building Ubuntu "golden" images with the Azure Image Builder (AIB). This guide focuses on automating the use of Azure Image Builder. If you want more information on how to use Azure Image Builder see the :doc:`Create a Pro golden image <create-pro-fips-golden-image>` guide which explains the individual steps.
+
+What you'll need
+----------------
+
+- A Microsoft Azure account
+- `Azure Command-Line Interface`_
+- A GitHub account
+- (Optional) Golang for local debugging/testing
+
+Set up your Azure credentials
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To authenticate with the Azure Go SDK you are going to use an Azure service principal with a client secret.
```

Comment:
Since Azure Go SDK is mentioned here for the first time, it is a little confusing. Could we add something like "The GitHub action uses Azure Go SDK to setup and run Azure Image Builder" at the beginning.